node_proof_of_concept.js

```diff
@@ -21,6 +18,8 @@ var pg = require('pg')
 var format = require('pg-format')
 var PGUSER = process.argv[2]
 var PGPASSWORD = process.argv[3]
+if(PGPASSWORD == '%')
+	PGPASSWORD = null;
```

Comment:
The indentation is kind of weird here and [here](https://github.com/lbraun/csr_lookup/pull/7/files#diff-626ca3cb86e13754a56fb5a86b7c0fa0L33). I think it's because you are using tabs instead of spaces. We can switch to tabs if you want, just let me know so we can keep it consistent. Can you updated it either way before we merge this PR? Gracias!